v3/src/components/case-table/inspector-panel/trash-menu-list.tsx

```diff
@@ -0,0 +1,75 @@
+import { MenuItem, MenuList } from "@chakra-ui/react"
+import React from "react"
+import { useDataSetContext } from "../../../hooks/use-data-set-context"
+import { IAttribute } from "../../../models/data/attribute"
+import t from "../../../utilities/translation/translate"
+
+export const TrashMenuList = () => {
+  const data = useDataSetContext()
+
+  const createGhostCase = (attrs:IAttribute[]) => {
+    const gCase:Record<string, any> = {}
+    attrs?.forEach(attr => {
+      gCase[attr.name]=""
+    })
+    return gCase
+  }
```

Comment:
Now may or may not be the time to deal with it, but this whole ghost case business is a kluge. We shouldn't be adding bogus data to the DataSet. Instead, the case table should be modified to deal with an empty DataSet properly. I let it go initially because it seemed expedient when we were just bootstrapping our way to an initial table state, but much of the code in this component just serves to further cement the kluge, so it may be time to address it properly. It should be the case table's job to render the DataSet in its current form rather than to modify the DataSet so it's easier to render.

Reply:
Rather than going through another round of changes/review I just went ahead and implemented what I had in mind. 😬